opendatahub-io/odh-manifests
# Add metrics port for modelmesh infra pods

Signed-off-by: Ricardo Martinelli de Oliveira <rmartine@redhat.com>

## Description
Add metrics port for modelmesh services

## How Has This Been Tested?
* Deploy modelmesh kfdef
* Configure prometheus to scrape metrics from the following service objects:
    * odh-model-controller-metrics-service
    * modelmesh-controlle
* Check if the targets are available

## Merge criteria:
